[2022-05-13 10:58:08,450] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Collect_Data_Dag.get_data_src_measures_taken_country_wise manual__2022-05-13T10:58:08.119207+00:00 [queued]>
[2022-05-13 10:58:08,456] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Collect_Data_Dag.get_data_src_measures_taken_country_wise manual__2022-05-13T10:58:08.119207+00:00 [queued]>
[2022-05-13 10:58:08,457] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 10:58:08,457] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-05-13 10:58:08,458] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 10:58:08,466] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): get_data_src_measures_taken_country_wise> on 2022-05-13 10:58:08.119207+00:00
[2022-05-13 10:58:08,471] {standard_task_runner.py:52} INFO - Started process 1702 to run task
[2022-05-13 10:58:08,474] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Collect_Data_Dag', 'get_data_src_measures_taken_country_wise', 'manual__2022-05-13T10:58:08.119207+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/collect_data_dag.py', '--cfg-path', '/tmp/tmpikr3g1xk', '--error-file', '/tmp/tmpdae7ej3n']
[2022-05-13 10:58:08,475] {standard_task_runner.py:80} INFO - Job 141: Subtask get_data_src_measures_taken_country_wise
[2022-05-13 10:58:08,517] {task_command.py:369} INFO - Running <TaskInstance: Collect_Data_Dag.get_data_src_measures_taken_country_wise manual__2022-05-13T10:58:08.119207+00:00 [running]> on host b558eea96bdb
[2022-05-13 10:58:08,723] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=Collect_Data_Dag
AIRFLOW_CTX_TASK_ID=get_data_src_measures_taken_country_wise
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T10:58:08.119207+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-13T10:58:08.119207+00:00
[2022-05-13 10:58:08,724] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-13 10:58:08,725] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/dags/scripts/dag_data_source_api_5.py']
[2022-05-13 10:58:08,732] {subprocess.py:85} INFO - Output:
[2022-05-13 10:58:12,262] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-13 10:58:12,282] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Collect_Data_Dag, task_id=get_data_src_measures_taken_country_wise, execution_date=20220513T105808, start_date=20220513T105808, end_date=20220513T105812
[2022-05-13 10:58:12,308] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-13 10:58:12,333] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
